sig-contributor-experience/README.md

```diff
@@ -68,7 +68,7 @@ Contributor Communications focuses on amplifying the success of Kubernetes contr
 - **Owners:**
   - [kubernetes-sigs/contributor-tweets](https://github.com/kubernetes-sigs/contributor-tweets/blob/master/OWNERS)
   - [kubernetes-sigs/lwkd](https://github.com/kubernetes-sigs/lwkd/blob/main/OWNERS)
-  - [kubernetes/community/communication/marketing-team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/OWNERS)
+  - [kubernetes/community/communication/contributor-comms](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/OWNERS)
```

Comment:
You'll have to make this change in sigs.yaml and run `make` in the root of the repo to reflect this change in the doc.

Reply:
Exact location to change: https://github.com/kubernetes/community/blob/master/sigs.yaml#L1361